instrumentation/p6spy/src/main/java/brave/p6spy/TracingP6SpyOptions.java

```diff
@@ -1,32 +1,58 @@
 package brave.p6spy;
 
-import com.p6spy.engine.spy.P6SpyOptions;
+import com.p6spy.engine.logging.P6LogLoadableOptions;
+import com.p6spy.engine.logging.P6LogOptions;
+import com.p6spy.engine.spy.P6LoadableOptions;
 import com.p6spy.engine.spy.option.P6OptionsRepository;
+
+import javax.management.StandardMBean;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
-final class TracingP6SpyOptions extends P6SpyOptions {
+final class TracingP6SpyOptions extends StandardMBean
+    implements P6LoadableOptions, TracingP6SpyOptionsMBean {
+
   static final String REMOTE_SERVICE_NAME = "remoteServiceName";
   static final String INCLUDE_PARAMETER_VALUES = "includeParameterValues";
+  private static final Map<String, String> defaults = new LinkedHashMap();
 
-  final P6OptionsRepository optionsRepository;
+  private final P6OptionsRepository optionsRepository;
+  private final P6LogLoadableOptions logLoadableOptions;
+
+  static {
+    defaults.put(INCLUDE_PARAMETER_VALUES, Boolean.FALSE.toString());
+  }
 
-  TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
-    super(optionsRepository);
+  public TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
+    super(TracingP6SpyOptionsMBean.class, false);
+    logLoadableOptions = new P6LogOptions(optionsRepository);
     this.optionsRepository = optionsRepository;
   }
 
   @Override public void load(Map<String, String> options) {
-    super.load(options);
+    logLoadableOptions.load(options);
     optionsRepository.set(String.class, REMOTE_SERVICE_NAME, options.get(REMOTE_SERVICE_NAME));
-    optionsRepository.set(Boolean.class, INCLUDE_PARAMETER_VALUES, options.get(INCLUDE_PARAMETER_VALUES));
+    optionsRepository.set(Boolean.class, INCLUDE_PARAMETER_VALUES,
+        options.get(INCLUDE_PARAMETER_VALUES));
+  }
+
+  @Override
+  public Map<String, String> getDefaults() {
+    Map<String, String> allDefaults = new LinkedHashMap();
+    allDefaults.putAll(defaults);
```

Comment:
inline`defaults` vs a static initializes for the hashmap which only has one attribute.